src/Microsoft.DotNet.ProjectModel/project.json

```diff
@@ -6,7 +6,7 @@
     "description": "Types to model a .NET Project",
     "dependencies": {
         "System.Reflection.Metadata": "1.2.0-rc2-23901",
-        "NuGet.Packaging": "3.4.0-beta-632",
+        "NuGet.Packaging": "3.4.0-rtm-beta-721",
```

Comment:
And BTW there are two problems:
1. RTM + beta = weird
2. The build number is 3-digits. You'll run out of build numbers in ~270 builds.